x-pack/plugins/encrypted_saved_objects/server/plugin.ts

```diff
@@ -97,12 +93,9 @@ export class Plugin {
 
   public start() {
     this.logger.debug('Starting plugin');
-
     return {
       isEncryptionError: (error: Error) => error instanceof EncryptionError,
-      getDecryptedAsInternalUser: (type: string, id: string, options?: SavedObjectsBaseOptions) => {
-        return this.savedObjectsSetup.getDecryptedAsInternalUser(type, id, options);
-      },
+      getClient: (includedHiddenTypes?: string[]) => this.savedObjectsSetup(includedHiddenTypes),
```

Comment:
__ask:__ sorry for being late here, but I have two minor asks:

* can you make parameter as `{ includedHiddenTypes?: string[] }` instead so that it's easier to understand the intention when you read consumer code (`getClient(['one', 'two'])` vs `getClient({ includedHiddenTypes: ['one', 'two'] })` and add new options in the future?

* would you mind updating `README.md` of the plugin to mention new API?

Thanks!

Reply:
Sorry @azasypkin a Github UI quirk made me miss this last minute comment!
I' can do that as part of a follow up PR that I'll be submitting today anyway- I'll ping you to review :)

Reply:
Thanks :+1: